IAMOTZ/ievents
# #152995325 Create template for admin transaction page

### What does this PR do?
Add the template for admin transaction-page
#### Description of Task to be completed?
Create template page for admin to see his transactions
 #### How should this be manually tested?
- clone the repo
- cd into the template directory
- open the transactions.html with any browser of your choice
- the template should open in that browser
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#152995325
#### Screenshots (if appropriate)
![justtnow-01](https://user-images.githubusercontent.com/26048536/32993032-a249adb2-cd52-11e7-9a92-ebae00e41dc7.png)
#### Questions:
None
